src/components/ProfileAvatar.tsx

```diff
@@ -0,0 +1,198 @@
+import Slider from '@mui/material/Slider'
+import VerifiedBorder from 'assets/VerifiedProfilePic.svg'
+import React, { useEffect, useState } from 'react'
+import { Button } from '@mui/material'
+import { Modal } from 'react-bootstrap'
+import Cropper from 'react-easy-crop'
+import { Area } from 'react-easy-crop/types'
+
+import { displayToast } from 'synapse-react-client/dist/containers/ToastMessage'
+import {
+  getFileHandleByIdURL,
+  updateMyUserProfile,
+  uploadFile,
+} from 'synapse-react-client/dist/utils/SynapseClient'
+import { useSynapseContext } from 'synapse-react-client/dist/utils/SynapseContext'
+import {
+  FileUploadComplete,
+  UserProfile,
+} from 'synapse-react-client/dist/utils/synapseTypes'
+import { getCroppedImg } from './CropImage'
+
+export type ProfileAvatarProps = {
+  userProfile?: UserProfile
+  verified?: boolean
+  onProfileUpdated: () => void
+}
+
+export const ProfileAvatar = (props: ProfileAvatarProps) => {
+  const { userProfile, verified, onProfileUpdated } = props
+  const { accessToken } = useSynapseContext()
+  const [profilePicUrl, setProfilePicUrl] = useState<string | undefined>()
+  const [image, setImage] = useState<string | undefined>()
+  const [croppedArea, setCroppedArea] = useState<Area | undefined>()
+  const [crop, setCrop] = useState({ x: 0, y: 0 })
+  const [zoom, setZoom] = useState(1)
+  const [openCropModal, setOpenCropModal] = useState(false)
+  const [imageLoading, setImageLoading] = useState(false)
+
+  useEffect(() => {
+    if (userProfile?.profilePicureFileHandleId) {
+      getFileHandleByIdURL(
+        userProfile?.profilePicureFileHandleId as string,
+        accessToken,
+      ).then(picUrl => {
+        setProfilePicUrl(picUrl)
+      })
+    }
+  }, [userProfile])
+
+  const updateUserProfile = async (newFileHandleId: string) => {
+    try {
+      if (userProfile) {
+        userProfile.profilePicureFileHandleId = newFileHandleId as string
+        await updateMyUserProfile(userProfile, accessToken)
+        displayToast('Profile picture has been successfully updated', 'success')
+        onProfileUpdated()
```

Comment:
In the old Profile page implementation, you could update your picture without updating your profile. This version updates immediately.  It then informs the AccountSettings page (in this case) that the Profile has been updated (which will cause it to refetch, and trickle down the new UserProfile object)